Doc/whatsnew/3.11.rst

```diff
@@ -502,6 +502,14 @@ time
   (Contributed by Benjamin Szőke, Dong-hee Na, Eryk Sun and Victor Stinner in :issue:`21302` and :issue:`45429`.)
 
 
+tkinter
+-------
+
+* Added method ``info_patchlevel()`` which returns the exact version of
+  the Tcl library as a named tuple similar to :data:`sys.version_info`.
+  (Contributed by Serhiy Storchaka in :issue:`91827`.)
```

Comment:
Azure pipelines docs build is currently failing because it should be ``:gh:`91827`` not ``:issue:`91827``

Reply:
Woops I didn't see that Serhiy already opened a PR #92395 for this.